src/ranking/effectiveness/channel.ts

```diff
@@ -39,7 +39,7 @@ export namespace TypeChannelScore {
     };
 
     [Q, T, TIMEUNIT_T].forEach((type) => {
-      keys(CONTINUOUS_TYPE_CHANNEL_SCORE).forEach((channel: Channel) => {
+      keys(CONTINUOUS_TYPE_CHANNEL_SCORE).forEach((channel) => {
```

Comment:
Not sure why you remove `: Channel` in this file.  Is this a result of incorrect merge conflict resolutions? If so, undo them.  :)